controllers/topology/internal/contract/controlplane.go

```diff
@@ -135,3 +242,87 @@ func (r *ControlPlaneReplicas) Set(obj *unstructured.Unstructured, value int64)
 	}
 	return nil
 }
+
+// ControlPlaneStatusReplicas is a helper struct for working on status.replicas of control plane.
+type ControlPlaneStatusReplicas struct{}
```

Comment:
@ykakarap If I see correctly the only difference between the following structs (and the one above) is the path. What do you think about de-duplicating it? (similar like we did with ref)